packages/storybook-addon/package.json

```
@@ -0,0 +1,70 @@
+{
+  "name": "@urql/storybook-addon",
+  "version": "1.0.3",
+  "description": "Make stories in Storybook with urql",
+  "sideEffects": false,
+  "homepage": "https://github.com/FormidableLabs/urql/tree/main/packages/storybook-addon",
+  "bugs": "https://github.com/FormidableLabs/urql/issues",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/FormidableLabs/urql.git",
+    "directory": "packages/storybook-addon"
+  },
+  "keywords": [
+    "addon",
+    "storybook",
+    "urql",
+    "formidablelabs",
+    "preact",
+    "react"
+  ],
+  "main": "dist/urql-storybook-addon.js",
+  "module": "dist/urql-storybook-addon.mjs",
+  "types": "dist/types/storybook.d.ts",
+  "source": "src/index.ts",
+  "files": [
+    "LICENSE",
+    "CHANGELOG.md",
+    "README.md",
+    "dist/"
+  ],
+  "exports": {
+    ".": {
+      "import": "./dist/urql-storybook-addon.js",
+      "require": "./dist/urql-storybook-addon.mjs",
+      "types": "./dist/types/decorator/react.d.ts",
+      "source": "./src/decorator/react.ts"
+    },
+    "./preact": {
+      "import": "./dist/urql-storybook-addon-preact.js",
+      "require": "./dist/urql-storybook-addon-preact.js",
+      "types": "./dist/types/decorator/preact.d.ts",
+      "source": "./src/decorator/preact.ts"
+    },
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "clean": "rimraf dist",
+    "check": "tsc --noEmit",
+    "lint": "eslint --ext=js,jsx,ts,tsx .",
+    "build": "rollup -c ../../scripts/rollup/config.js && cp src/global.d.ts dist/global.d.ts",
+    "prepublishOnly": "yarn clean && yarn build"
```

```suggestion
    "build": "rollup -c ../../scripts/rollup/config.js",
    "postbuild": "cp src/global.d.ts dist/global.d.ts",
    "prepublishOnly": "run-s clean build"
```

Just to make this work on Windows